src/scripts/main.js

```diff
@@ -1 +1,21 @@
 'use strict';
+
+document.addEventListener('DOMContentLoaded', () => {
+  const mainImage = document.querySelector('#largeImg');
+
+  const thumbnails = document.querySelector('#thumbs');
+
+  function changeImage(e) {
+    const clickedImage = e.target.tagName === 'IMG' ? e.target : null;
```

Comment:
Here you're assuming that the clicked element will always be an image. But what if the user clicks on the link but outside of the image? In that case, `e.target` will be the link, not the image, and `clickedImage` will be null. It's better to always take `e.target.closest('a')` and then get the image from it, if it exists.